recipes/rstudio/meta.yaml

```diff
@@ -0,0 +1,173 @@
+{% set version = "2022.02.2" %}
+
+package:
+  name: rstudio
+  version: {{ version }}
+
+source:
+  url: "https://github.com/rstudio/rstudio/archive/refs/tags/v{{ version }}+485.tar.gz"
```

Comment:
I think the +485 needs to go into the version string, as it will change (its the build count on their dailies before they get a release).

Reply:
https://github.com/conda-forge/rstudio-feedstock/pull/1

Reply:
Thank you!